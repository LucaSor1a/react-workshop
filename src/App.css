.vehicle__item {
  margin-bottom: 2em;
}
