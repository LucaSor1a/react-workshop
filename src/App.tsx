import Layout from "components/layout/Layout";
import React from "react";
import VehicleList from "components/vehicles/List";
import "bootstrap/dist/css/bootstrap.min.css";
import "./App.css";

function App() {
  return (
    <Layout>
      <VehicleList />
    </Layout>
  );
}

export default App;
