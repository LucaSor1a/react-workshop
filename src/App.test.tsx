import React from "react";
import { render } from "@testing-library/react";
import App from "./App";

jest.mock("components/vehicles/List", () => () => (
  <div data-testid="vehicle-list" />
));

test("Renders without crashing", () => {
  const { container, getByText, getByTestId } = render(<App />);

  expect(getByText("Autos Online")).toBeInTheDocument();
  expect(getByTestId("vehicle-list")).toBeDefined();

  expect(container).toMatchSnapshot();
});
