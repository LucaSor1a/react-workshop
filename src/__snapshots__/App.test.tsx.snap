// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders without crashing 1`] = `
<div>
  <div
    class="container-fluid"
  >
    <div
      class="row"
    >
      <h1>
        Autos Online
      </h1>
    </div>
  </div>
  <div
    class="container-fluid"
  >
    <div
      class="row"
    >
      <div
        data-testid="vehicle-list"
      />
    </div>
  </div>
  <div
    class="container-fluid"
  >
    <div
      class="row"
    >
      Copyright 
      2020
    </div>
  </div>
</div>
`;
